src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -213,7 +147,14 @@ function MoneyRequestPreview({
         }
 
         let message = translate('iou.cash');
-        if (ReportUtils.isPaidGroupPolicyExpenseReport(iouReport) && ReportUtils.isReportApproved(iouReport) && !ReportUtils.isSettled(iouReport?.reportID)) {
+        if (hasViolations && transaction) {
+            const violations = TransactionUtils.getTransactionViolations(transaction, transactionViolations);
```

Comment:
Similar to the comment above, can we pass `transaction.transactionID` here, and revert the change to `TransactionUtils. getTransactionViolations ` so it keeps accepting only a transactionID?

Reply:
Then the call would need to be: 

```
const hasViolations = TransactionUtils.hasViolation(transaction?.transactionID ?? '', transactionViolations);
```

which is far more complicated, and will be required every time someone wants to use this.

Reply:
Because OnyxEntry is a nullable type.